src/client.ts

```diff
@@ -1,3 +1,6 @@
+import * as EventEmitter from 'events';
+import StrictEventEmitter from 'strict-event-emitter-types';
```

Comment:
There are a few alternatives to this, but they are all reimplementations of EventEmitter.  Using strict-event-emitter uses fancy typescript (https://medium.com/@bterlson/strongly-typed-event-emitters-2c2345801de8) with the implementation backed by popular events (or eventemitter3) package for the actual eventing.

Reply:
Removed this library in favor of our own interface definition.  This allows us to have a bit more control over the types of emits/on's that we provide.